frontend/src/app/main/ui/auth/register.cljs

```diff
@@ -18,19 +18,68 @@
    [app.main.ui.auth.login :as login]
    [app.main.ui.components.forms :as fm]
    [app.main.ui.components.link :as lk]
-   [app.main.ui.icons :as i]
    [app.util.i18n :as i18n :refer [tr]]
    [app.util.storage :as storage]
    [beicon.v2.core :as rx]
    [rumext.v2 :as mf]))
 
 ;; --- PAGE: Register
 
+(mf/defc newsletter-options
```

Comment:
New components should follow the new call convention for react components (name ends with `*`, no need for explicit `{::mf/props :obj}`, is now a defaut with components with `*` in the name, and use `[:>` instead of `[:&` for instantiate it)